src/utils/sentry.js

```diff
@@ -0,0 +1,41 @@
+const { default: Toucan } = require('toucan-js');
+
+// Thanks @cloudflare/worker-sentry for the base of this
+module.exports.initSentry = (event, env, opts = {}) => {
+    // Use Toucan to interact with Sentry
+    const sentry = new Toucan({
+        dsn: env.SENTRY_DSN,
+        event: event,
+        allowedHeaders: [
+            'user-agent',
+            'cf-challenge',
+            'accept-encoding',
+            'accept-language',
+            'cf-ray',
+            'content-length',
+            'content-type',
+            'x-real-ip',
+            'host',
+        ],
+        allowedSearchParams: /(.*)/,
+        rewriteFrames: {
+            root: '/',
+        },
+        ...opts,
+    });
+
+    // Get the request
+    const request = event.request;
+
+    // Determine with Cloudflare colo the req went to
+    const colo = request.cf && request.cf.colo ? request.cf.colo : 'UNKNOWN';
```

Comment:
```suggestion
    const colo = request?.cf?.colo || 'UNKNOWN';
```
Once #20 is in, this can use optional chaining.